arangod/Aql/RestAqlHandler.cpp

```diff
@@ -752,12 +758,17 @@ RestStatus RestAqlHandler::handleUseQuery(std::string const& operation,
 
     auto result = AqlExecuteResult{state, skipped, std::move(items)};
     answerBuilder.add(VPackValue(StaticStrings::AqlRemoteResult));
-    result.toVelocyPack(answerBuilder, &_engine->getQuery().vpackOptions());
+    result.toVelocyPack(answerBuilder, opts);
     answerBuilder.add(StaticStrings::Code, VPackValue(TRI_ERROR_NO_ERROR));
   } else if (operation == "initializeCursor") {
+    // TODO: remove the following 2 lines if the assertion holds true
     auto pos = VelocyPackHelper::getNumericValue<size_t>(querySlice, "pos", 0);
+    TRI_ASSERT(pos == 0);
     Result res;
+    // TODO: remove the following if condition and blcok if the assertion
```

Comment:
```suggestion
    // TODO: remove the following if condition and block if the assertion
```

Reply:
I did not use your suggestion, but instead resolved the TODO and removed the unreachable code.